src/inc/msquic.h

```diff
@@ -485,6 +485,8 @@ typedef struct QUIC_STATISTICS_V2 {
 
     uint32_t DestCidUpdateCount;            // Number of times the destionation CID changed.
 
+    uint64_t RecvGreaseBitTpCount;          // Count of received grease transport parameter from the peer.
```

Comment:
No, we don't need this. I was more saying we should have a flag at the top that says `GreaseBitNegotiated`.

Reply:
Oh, something like I added to stats? `FixedBitNegotiated`?